docs/admin/pool_operations.md

```diff
@@ -155,7 +155,7 @@ Pool created with 6.00% SCM/NVMe ratio
 
 This created a pool with UUID 8a05bf3a-a088-4a77-bb9f-df989fce7cc8,
 with pool service redundancy enabled by default
-(pool service replicas on ranks 1-3).
+(pool service replicas on ranks 1-5).
 
 If no redundancy is desired, use --nsvc=1 in order to specify that only
 a single pool service replica should be created.
```

Comment:
Yes. I made such a change initially, but discarded it after deciding to not to deprecate "--nsvc" in this patch (to avoid having to update ftest). On second thought it makes sense, even without the deprecation, to advertise "svc_rf" more and "--nsvc" less. Will do if the PR needs another refresh.